docker-compose/is/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '2.3'
+services:
+  mysql:
+    image: mysql:5.7.28
```

Comment:
@ThilinaManamgoda shall we update the image to the latest `5.7.29` version?

Reply:
+1

Reply:
done